### PR TITLE
Create the marketing layout

### PR DESCRIPTION
**Pull request by shadikhani**

### CODE REVIEW

**Comment by tmattsso** [comment]
The menu bar can be removed, it isn't relevant for the demo.

The wrapping should work a bit better; if 3 items can't fit side-by side, then the items should rearrange (as it dos now). but each item should take the max width they can; you can add flex-grow to each layout and removing the spacing for the main horizontallayout (to fix padding).

contentLayout needs a better name

boxContainer needs a better name

        // It removes extra space above the contentLayout.
        contentLayout.getStyle().set("margin", "0");
this can be removed once the menubar is removed.

refactor the creation of box1, 2 and 3 to a factory method that returns a Component. The method receives the header and content strings.

Also, this branch needs to be merged to master.

File: src/main/java/com/vaadin/demo/views/MarketingFormView.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+        Component card1 = creadteCard(cardHeader, cardContent);
+        Component card2 = creadteCard(cardHeader, cardContent);
+        Component card3 = creadteCard(cardHeader, cardContent);
+
+        container.add(card1, card2, card3);
+        add(layout, container);
+    }
+
+    private Component creadteCard(String cardHeader, String cardContent) {
```
Comment (tmattsso) [inline src/main/java/com/vaadin/demo/views/MarketingFormView.java]: typo in name :)

File: frontend/src/views/intro-view.js [hunk 2]
```diff
@@ -106,6 +106,10 @@ class IntroView extends PolymerElement {
                         <a slot="title" router-link href="checkout-form">Checkout form</a>
                         <p>A checkout form including a billing address and summary of shopping card</p>
                     </ex-card>
+                    <ex-card>
+                        <a slot="title" router-link href="marketing">Marketing form</a>
+                        <p>A marketing form including a main part and three more parts under the main part</p>
```
Comment (tmattsso) [inline frontend/src/views/intro-view.js]: ..three responsive article cards under..

File: src/main/java/com/vaadin/demo/views/MarketingFormView.java (hunk 1 above)
Comment (shadikhani) [reply src/main/java/com/vaadin/demo/views/MarketingFormView.java]: Done.

File: frontend/src/views/intro-view.js (hunk 2 above)
Comment (shadikhani) [reply frontend/src/views/intro-view.js]: Done.

**Pull request merged**